### PR TITLE
feat: add tracking to main header items

### PR DESCRIPTION
**Pull request by talamcol**
References https://smg-au.atlassian.net/browse/DM-3049

## Motivation and context

To measure the performance of the new EV hub, we add tracking to the main navigation items in the header to have a baseline.

FYI: as per Raluca, at the moment we only track the main items. The ones collapsed in drawers not.

## Before

No tracking

## After

Tracking

### CODE REVIEW

File: src/components/navigation/header/config/headerLinks.ts [hunk 1]
```diff
@@ -88,6 +106,12 @@ export const headerLinks: NavigationLinkConfigProps[] = [
         motoscout24: true,
       },
     },
+    onClick: () =>
+      trackEvent?.({
+        eventCategory: navigationEventCategory,
+        eventAction: 'sell',
+        eventLabel: 'professional',
```
[the remark this answers is not in the document]
Comment (talamcol) [reply src/components/navigation/header/config/headerLinks.ts:113]: yeah she doesn't care about that. Removed it 😅

File: src/components/navigation/header/config/DrawerNodeItems.tsx [hunk 2]
```diff
@@ -1402,10 +1391,15 @@ export const drawerNodeItems = ({
               professional: true,
             },
             brand: {
-              autoscout24: false,
+              autoscout24: true,
```
Comment (talamcol) [inline src/components/navigation/header/config/DrawerNodeItems.tsx:1394]: this was wrong. it is different on the desktop version

File: src/components/navigation/header/config/DrawerNodeItems.tsx [hunk 3]
```diff
@@ -1379,13 +1363,18 @@ export const drawerNodeItems = ({
           visibilitySettings: {
             userType: {
               private: true,
-              professional: true,
+              professional: false,
```
Comment (talamcol) [inline src/components/navigation/header/config/DrawerNodeItems.tsx:1366]: this was wrong. it is different on the desktop version

**Review by GutierrezGo — APPROVED** [review]
Code looks good to me.
Thank you for checking with the PM about the eventLabel

**Pull request merged**